packages/story-editor/src/components/panels/design/textAccessibility/textAccessibility.js

```diff
@@ -98,11 +84,49 @@ function TextAccessibilityPanel({ selectedElements, pushUpdate }) {
     const newTags = getTextElementTagNames(selectedTextElements, value);
     setCurrentTags(Array.from(newTags.values()));
   };
+
   // Check if tags match for selected item in dropdown
   // If they don't match, we want to show 'Mixed'
   // However, if they match, show the matching value
   const currentValue =
-    new Set(currentTags || []).size === 1 ? currentTags[0] : '((MULTIPLE))';
+    new Set(currentTags || []).size === 1 ? currentTags[0] : MULTIPLE_VALUE;
+
+  const isIndeterminate = MULTIPLE_VALUE === currentValue;
+
+  const tagName = getCommonValue(selectedElements, 'tagName', 'auto');
+
+  const selectedValue = 'auto' === tagName ? 'auto' : currentValue;
+
+  const options = [
+    {
+      label:
+        'auto' === selectedValue
+          ? sprintf(
+              /* translators: %s: heading level. */
+              __('Automatic (%s)', 'web-stories'),
+              HEADING_LEVELS[currentValue]
+            )
+          : __('Automatic', 'web-stories'),
+      value: 'auto',
+    },
+    {
+      label: HEADING_LEVELS.h1,
+      value: 'h1',
+    },
+    {
+      label: HEADING_LEVELS.h2,
+      value: 'h2',
+    },
+    {
+      label: HEADING_LEVELS.h3,
+      value: 'h3',
+    },
+    {
+      label: HEADING_LEVELS.p,
+      value: 'p',
+    },
```

Comment:
This could just be:

```js
    ...Object.entries(HEADING_LEVELS).map(([value, label]) => ({ value, label })),
```

Not the easiest to read, but it would work and would not require updates if we were ever to add more element types 🤷‍♂️ .